### DISCUSSION TITLE
Self-built, relay. Connected, waiting for screen transfer. The screen does not display, how to solve it?

### DISCUSSION THREAD

**Discussion by wgf4242** [rustdesk/rustdesk#4493] [General]
配置正常。连接正常就是没画布。
传输文件是没问题。

有个情况。如果用向日葵连接了主机，rustdesk远程连接就能连上了。不知道什么原因。
但是关了向日葵，rustdesk 就会断开。

防火墙已经关闭。

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/releases/nightly
if nightly still does not work, please show us your monitor devices in device manager when connected with sunflower.

**Reply by rustdesk** [reply]
@fufesou

**Reply by jeeminrose** [reply]
![image](https://github.com/user-attachments/assets/5a9cedfa-2aa3-4d0b-bfdb-ee62c1fbfeed)

**Reply by rustdesk** [reply]
@jeeminrose you can aslo contact us with email, c3VuYm9lYXN5QGdtYWlsLmNvbQo=, base64. Our dev will debug this issue with you.

**Reply by rustdesk** [reply]
> @jeeminrose you can aslo contact us with email, c3VuYm9lYXN5QGdtYWlsLmNvbQo=, base64. Our dev will debug this issue with you.

@21pages

**Comment by rustdesk** [comment]
Please check this, https://github.com/rustdesk/rustdesk/discussions/9124#discussioncomment-10391439

**Comment by rustdesk** [comment]
close and lock to favor https://github.com/rustdesk/rustdesk/discussions/9124#discussioncomment-10391439
